Snippets/Core/Core_6/Routing/RoutingAPIs.cs

```diff
@@ -117,6 +118,15 @@ void DynamicEndpointMapping(EndpointConfiguration endpointConfiguration)
             #endregion
         }
 
+        void CustomDistributionStrategy(EndpointConfiguration endpointConfiguration)
+        {
+            #region Routing-CustomDistributionStrategy
+
+            var routing = endpointConfiguration.UnicastRouting();
+            routing.Mapping.SetMessageDistributionStrategy(new CustomStrategy(), messageType => messageType.GetInterfaces().Contains(typeof(IUseCustomDistributionStrategy)));
```

Comment:
not saying u need the reflection, since that might make the sample overly complex. i think a note would suffice